src/Gui/DlgUnitsCalculatorImp.cpp

```diff
@@ -86,27 +88,34 @@ DlgUnitsCalculator::DlgUnitsCalculator( QWidget* parent, Qt::WindowFlags fl )
           << Base::Unit::ElectricCharge
           << Base::Unit::ElectricCurrent
           << Base::Unit::ElectricPotential
-          << Base::Unit::Frequency
           << Base::Unit::Force
-          << Base::Unit::Stiffness
+          << Base::Unit::Frequency
           << Base::Unit::HeatFlux
+          << Base::Unit::InverseArea
+          << Base::Unit::InverseLength
+          << Base::Unit::InverseVolume
+          << Base::Unit::KinematicViscosity
           << Base::Unit::Length
           << Base::Unit::LuminousIntensity
           << Base::Unit::Mass
           << Base::Unit::MagneticFieldStrength
           << Base::Unit::MagneticFlux
           << Base::Unit::MagneticFluxDensity
           << Base::Unit::Magnetization
-          << Base::Unit::Pressure
```

Comment:
Why did you delete this?

Reply:
I didn't; I moved it two lines down. Almost all the other quantities are in alphapetical order here and this one was slightly out of position.